packages/module/src/index.ts

```diff
@@ -1 +1,2 @@
+export * from './DeleteModal';
 export * from './ExtendedButton';
```

Comment:
I can put up a PR to delete the extendedButton. That's what comes with the extension seed, right? so we don't need it?

Reply:
It's ok, I have deleted it as part of this PR, so we don't have to handle conflicts